src/base/projects/Issue/IssueComment.svelte

```diff
@@ -18,11 +18,14 @@
   let profile: Profile | null = null;
 
   onMount(async () => {
-    if (comment.author.identity.ens?.name) {
+    if (comment.author.identity?.ens?.name) {
       profile = await Profile.get(comment.author.identity.ens.name, ProfileType.Minimal, config);
     }
   });
 
+  $: source = profile?.avatar || comment.author.identity?.urn || comment.author.urn || comment.author.peer;
+  $: title = profile?.name || comment.author.identity?.ens?.name || comment.author.urn || comment.author.peer;
```

Comment:
Which part here is reactive? ie. what do we expect to be updated after the component is loaded and why?

Reply:
The `Profile.get` function is being called `onMount` so there is eventually profile data we want to use as the `source` and `title` of the user avatar.